grafana/provisioning/dashboards/monitoring.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+providers:
+- name: 'monitoring'
+  orgId: 1
+  folder: 'Monitoring'
+  type: file
+  disableDeletion: true
+  updateIntervalSeconds: 300
+  options:
+    path: /etc/grafana/provisioning/dashboards/monitoring
+
+
```

Comment:
This yaml will create the `Monitoring` folder.
All the json files included at the folder `/etc/grafana/provisioning/dashboards/monitoring` will be provisioned automatically at the `Monitoring` folder.

So if you want to add new dashboards, you just gotta add their corresponding JSONs files at this directory.